x-pack/plugins/security_solution/public/management/pages/event_filters/view/event_filters_list.tsx

```diff
@@ -114,6 +114,12 @@ const EVENT_FILTERS_PAGE_LABELS: ArtifactListPageProps['labels'] = {
       defaultMessage: '"{itemName}" has been removed from event filters list.',
       values: { itemName },
     }),
+  emptyStateTitleNoEntries: i18n.translate(
+    'xpack.securitySolution.eventFilters.emptyStateTitleNoEntries',
+    {
+      defaultMessage: 'There are no event filter entries to display.',
```

Comment:
Should this be `There are no event filters to display.` instead?

Reply:
🦅 👁️ !

Thanks for catching this! Here's the modification: d080743